idaes/models/unit_models/heat_exchanger_1D.py

```diff
@@ -430,96 +430,125 @@ def build(self):
 
         # Set flow directions for the control volume blocks and specify
         # discretization if not specified.
-        if self.config.flow_type == HeatExchangerFlowPattern.cocurrent:
-            set_direction_hot = FlowDirection.forward
-            set_direction_cold = FlowDirection.forward
+        if self.config.hot_side.transformation_method is useDefault:
+            _log.warning(
```

Comment:
Should this be a warning level message? I am of two minds, but I think this would be better as info.

Reply:
This was originally a warning-level message before my proposed revisions. I'd be happy to downgrade it to "Caution" or "Info".

Reply:
Maybe make it a caution then - it is definitely not a warning but probably warrants more than info.